src/main/java/org/apache/maven/plugins/changes/ChangesReport.java

```diff
@@ -75,6 +75,21 @@ public class ChangesReport extends AbstractChangesReport {
     @Parameter(property = "changes.addActionDate", defaultValue = "false")
     private boolean addActionDate;
 
+    /**
+     * Whether the change description should be passed as raw text to the Doxia sink or not.
```

Comment:
Whether the change description should be escaped in the generated report. Set to false if the change descriptions already contain escape sequences such as &lt; or \"

You shouldn't mention Doxia here. That is an implementation detail that shouldn't appear in user docs.

Reply:
that was the exact wording proposed by @michael-o (see https://github.com/apache/maven-changes-plugin/pull/113#discussion_r1973055924) - but i can change it again.